src/features/utils/lazy-feature-loader.test.js

```diff
@@ -0,0 +1,37 @@
+import { faker } from '@faker-js/faker'
+import { FEATURE_NAMES } from '../../loaders/features/features'
+import { lazyFeatureLoader } from './lazy-feature-loader'
+
+// Use enableAutomock to make it easier to mock all the things that get imported by the aggregators
+jest.enableAutomock()
```

Comment:
Is the purpose of this to prevent executing lazy-loaded aggregate modules?

Reply:
Pretty much. If you run just this test file with coverage, you shouldn't get any coverage on the feature aggregates. Mocking in these tests was a bit of trial and error.